node/src/BaseClient.ts

```diff
@@ -3015,6 +3016,36 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Returns the number of members in the sorted set stored at <code>key</code> with scores between
+     * 'minLex' and 'maxLex'.
+     *
+     * See https://valkey.io/commands/zlexcount/ for more details.
+     *
+     * @param key - The key of the sorted set.
+     * @param minLex - The minimum lex to count from. Can be positive/negative infinity, or specific lex and inclusivity.
+     * @param maxLex - The maximum lex to count up to. Can be positive/negative infinity, or specific lex and inclusivity.
+     * @returns The number of members in the specified lex range.
+     * - If 'key' does not exist, it is treated as an empty sorted set, and the command returns '0'.
+     * - If maxLex less than minLex, '0' is returned.
```

Comment:
nit: some minor suggestions to align with other docs
```suggestion
     * If 'key' does not exist, it is treated as an empty sorted set, and the command returns '0'.
     * If maxLex is less than minLex, '0' is returned.
```